plugins/modules/zos_job_output.py

```diff
@@ -80,7 +80,7 @@
 RETURN = r"""
 jobs:
   description:
-      List of jobs output.
+      List of jobs output, or a single NOTFOUND record.
```

Comment:
Maybe something like this:
description:
     List of the jobs output or a single NOTFOUND record when the job has not been run. 
     
     
`NOTFOUND` could change depending on my earlier comments

Reply:
I can see that

Reply:
I changed the entries to:
    Returns:
        list[dict] -- The output information for a list of jobs matching specified criteria.
        If no job status is found, this will return an empty job code with msg=Job not found

Let me know if that needs more tweaking

Reply:
Looks good , thanks @richp405